database-commons/src/main/java/io/cdap/plugin/db/batch/source/AbstractDBSource.java

```diff
@@ -296,6 +278,8 @@ private String getJDBCPluginId() {
 
   protected abstract String createConnectionString(String host, Integer port, String database);
 
+  protected abstract String createConnectionString();
```

Comment:
since everything needs to implement this now, can the other version that takes host, port, database be removed?

Reply:
Yep, doesn't need it anymore. Removed